designs/resourceCache/design.md

```diff
@@ -0,0 +1,155 @@
+# AWS Resource Cache
+
+List and describe calls to AWS services are used to fetch information about the resources 
+available within a customers account. Typical examples are:
+ - populate AWS explorer views
+ - populate resource-selection drop-down boxes
+
+In some cases the lifecycle of these describe calls is not controlled by the plugin but rather
```

Comment:
Is your point that we don't have control over when the IntelliJ refreshes the data? (This could be more clear.)

Reply:
Added abit more